### PR TITLE
KEYCLOAK-19486 Verify the WebAuthn registration functionality

### PR DESCRIPTION
**Pull request by mabartos**
JIRA: [KEYCLOAK-19486](https://issues.redhat.com/browse/KEYCLOAK-19486)

Core tests for WebAuthn registration. Some of those tests are ignored and I need to resolve them. However, it'd be great to have those tests included in testsuite in order to provide some template for WebAuthn tests for community members.

@mposolda @miquelsi Could you please take a look at it? Thanks.

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@mabartos LGTM, Thanks for these tests!

I can merge this once Miquel approves. I am not sure if we need pipeline run for this or not (Leaving this to you). Also not sure if you prefer this before or after https://github.com/keycloak/keycloak/pull/9029 or if order of PRs does not matter?

**Comment by mabartos** [comment]
@mposolda Ok, I removed the error message from WebAuthnAuthenticator.

**Comment by mabartos** [comment]
I'd say we don't need pipeline job for each PR... I'll do the polishing lately anyway. The order of the PRs doesn't matter.

**Comment by mposolda** [comment]
@mabartos Ok, Thanks. Waiting for @miquelsi approve for this.

**Comment by mabartos** [comment]
Resolved conflicts.

**Review by miquelsi — APPROVED** [review]
LGTM. Thanks Martin! Please @mposolda merge it when you have a chance.

**Review by mposolda — APPROVED** [review]
@mabartos @miquelsi Thanks!

**Pull request merged**
